src/eko/io/runcards.py

```diff
@@ -0,0 +1,373 @@
+"""Structures to hold runcards information.
+
+All energy scales in the runcards should be saved linearly, not the squared
+value, for consistency.
+Squares are consistenly taken inside.
+
+"""
+from dataclasses import dataclass
+from math import nan
+from typing import Optional, Union
+
+import numpy as np
+import numpy.typing as npt
+
+from .. import basis_rotation as br
+from .. import interpolation
+from .. import version as vmod
+from .dictlike import DictLike
+from .types import (
+    CouplingsRef,
+    EvolutionMethod,
+    FlavorsNumber,
+    HeavyQuarkMasses,
+    IntrinsicFlavors,
+    InversionMethod,
+    MatchingScales,
+    Order,
+    QuarkMassSchemes,
+    RawCard,
+    ScaleVariationsMethod,
+    T,
+)
+
+
+@dataclass
+class TheoryCard(DictLike):
+    """Represent theory card content."""
+
+    order: Order
+    """Perturbatiive order tuple, ``(QCD, QED)``."""
+    couplings: CouplingsRef
+    """"""
+    num_flavs_init: Optional[FlavorsNumber]
+    r"""Number of active flavors at fitting scale.
+
+    I.e. :math:`n_{f,\text{ref}}(\mu^2_0)`, formerly called ``nf0``.
+
+    """
+    num_flavs_max_pdf: FlavorsNumber
+    """"""
+    intrinsic_flavors: IntrinsicFlavors
+    """"""
+    quark_masses: HeavyQuarkMasses
+    """"""
+    quark_masses_scheme: QuarkMassSchemes
+    """Scheme used to specify heavy quark masses."""
+    matching: MatchingScales
+    """"""
+    # TODO: drop this
```

Comment:
I guess the `TODO` can be dropped - since we need the setting ... instead, we need a docstring

Reply:
Yes, instead of the dropping we might need a better name. If you have sensible proposals better, otherwise we will keep going with it.

Reply:
not for the moment, let's keep it for now